packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -279,7 +279,8 @@ abstract class RawKeyEvent extends Diagnosticable {
       case 'linux':
         data = RawKeyEventDataLinux(
             keyHelper: KeyHelper(message['toolkit'] ?? ''),
-            codePoint: message['codePoint'] ?? 0,
+            codePoint:
+                message['codePoint'] ?? '',
```

Comment:
trivial nit: looks like the formatting is off here

Reply:
Not sure what happened that this got out of sync. I just pushed again and now it's fixed